applications/FluidDynamicsApplication/custom_utilities/fluid_post_process_utilities.h

```diff
@@ -0,0 +1,156 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:      BSD License
+//                Kratos default license: kratos/license.txt
+//
+//  Main authors: Miguel Angel Celigueta
+//                Aditya Ghantasala
+//
+
+#ifndef POST_PROCESS_UTILITIES_H
+#define POST_PROCESS_UTILITIES_H
+
+#include "includes/define.h"
+#include "includes/variables.h"
+#include "includes/model_part.h"
+
+namespace Kratos
+{
+  ///@addtogroup FluidDynamicsApplication
+  ///@{
+
+  ///@name Kratos Globals
+  ///@{
+
+  ///@}
+  ///@name Type Definitions
+  ///@{
+
+  ///@}
+  ///@name  Enum's
+  ///@{
+
+  ///@}
+  ///@name  Functions
+  ///@{
+
+  ///@}
+  ///@name Kratos Classes
+  ///@{
+  ///
+  /** Auxiliary utility for computing different values as post processing.
+   */
```

Comment:
To me it looks ok but this should be a namespace and not a class (@rubenzorrilla we should really discuss this asap this is blocking too many things)

Reply:
we discussed this in the @KratosMultiphysics/technical-committee (highly controversial ;) )
in a nutshell, we will stick with classes for the moment